packages/nhost-js/src/core/nhost-client.ts

```diff
@@ -6,9 +6,21 @@ import { NhostGraphqlClient } from '../clients/graphql'
 
 export interface NhostClientConstructorParams extends Omit<NhostAuthConstructorParams, 'url'> {
   /**
-   * Nhost backend URL and Region.
+   * Nhost backend URL
+   * Will be deprecated in favor of `subdomain` and `region`
+   */
+  backendUrl?: string
+
+  /**
+   * App subdomain (e.g., ifieniwenfiwenwng)
+   * Use `localhost` in development
+   */
+  subdomain?: string
```

Comment:
ideally the typings should be review so we require either  `{ backendUrl }` or `{ subdomain, region }`, but not both, for instance in the way we do it for `nhost.auth.signIn`

Reply:
added a bunch of types :+1: